antenna/breakpad_resource.py

```diff
@@ -124,6 +136,9 @@ def __init__(self, config):
         # Queue for crashmover of crashes to save
         self.crashmover_save_queue = deque()
 
+        # Queue for publishing crashids to processor
+        self.crashmover_publish_queue = deque()
```

Comment:
This is effectively a two-step crashmoving process now. I thought about generalizing that to an n-step process but decided to wait on that until it's 3+-steps.